### PR TITLE
Added Card

### PR DESCRIPTION
**Pull request by dubeyaditya29**
Added different Card formats. 

https://user-images.githubusercontent.com/48158220/153673074-1f9404df-4603-4f09-89d0-2d69a85e4d0d.mp4

### CODE REVIEW

File: Stylesheet/variable.css [hunk 1]
```diff
@@ -72,4 +72,6 @@ body {
   --button-color-default: #1e3d58;
   --button-color-secondary: #bf9270;
   --text-bold: 600;
+  --card-padding: 0 0 0.5rem 0.4rem;
```
Comment (hinanshisuthar) [inline Stylesheet/variable.css]: You could use px unit for values <=10 px.

**Review by hinanshisuthar — COMMENTED** [review]
Suggested changes. Because you don't want responsiveness for very small units.  Everything else looks good including using of variables wherever possible.

File: Stylesheet/variable.css [hunk 2]
```diff
@@ -36,7 +36,7 @@ body {
 .code-snippet {
   border: 2px solid var(--color-grey);
   padding: 1rem;
-  width: 80%;
+  min-width: 31.25rem;
```
Comment (Hamdrive) [inline Stylesheet/variable.css]: Consider using a rounded value of width

File: components/Card/card.css [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+.card {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  width: 22rem;
+  border: 2px solid black;
+  border-radius: 1rem;
+}
+.card-image img {
```
Comment (Hamdrive) [inline components/Card/card.css:14]: Consider formatting your code, including spacing for better readability

File: components/Card/card.css [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+  height: 3rem;
+  border-radius: 3px 0 3px 0;
+  line-height: 2;
+  font-size: 20px;
+  color: var(--color-white);
+  font-weight: var(--text-bold);
+}
+.card-title {
+  background-color: var(--color-grey);
+  padding: 0 0 8px 9.6px;
```
Comment (Hamdrive) [inline components/Card/card.css]: Here as well, consider using rounded values

**Review by Hamdrive — COMMENTED** [review]
Great work done, components are simple and done well.
Will suggest the fixes mentioned, along with also providing spacing between the various sections of the cards.

File: Stylesheet/variable.css (hunk 1 above)
Comment (dubeyaditya29) [reply Stylesheet/variable.css]: Thanks I have actually removed that attribute will keep that in mind.

**Pull request merged**